rabbitmq/check.py

```diff
@@ -320,13 +320,22 @@ def get_connections_stat(self, instance, base_url, object_type, vhosts, custom_t
                               ssl_verify=ssl_verify, proxies=instance_proxy)
 
         stats = {vhost: 0 for vhost in vhosts}
+        connection_states = {}
         for conn in data:
+            if conn['state'] not in connection_states:
+                connection_states[conn['state']] = 0
+
+            connection_states[conn['state']] += 1
```

Comment:
Also, could you move this inside the condition block line 330. This will allow the integration to only count connection belonging to the list of vhosts specified by the user ?